### PR TITLE
Add raft backend initial support

### PR DESCRIPTION
**Pull request by securez**
I also commited a fix to a task that in my tests always report as chaged, this can be on another PR if you want.

### CODE REVIEW

**Comment by bbaassssiiee** [comment]
Please omit the commit for `fix: Avoid changed task on version check`.

File: templates/vault_backend_raft.j2 [hunk 1]
```diff
@@ -0,0 +1,4 @@
+storage "raft" {
+  path = "{{ vault_data_path }}"
+  node_id = "{{ vault_raft_node_id }}"
+}
```
Comment (bbaassssiiee) [inline templates/vault_backend_raft.j2]: Missing newline

**Comment by jpiron** [comment]
For the version check I might have the same issue : https://github.com/ansible-community/ansible-vault/pull/119#issuecomment-602571715

File: templates/vault_backend_raft.j2 (hunk 1 above)
Comment (seb-braun) [reply templates/vault_backend_raft.j2]: raft_node_id can't be the same in a multinode cluster.

**Comment by securez** [comment]
Sorry for the delay, I'm a bit busy this days, I already removed the check fix from the pull request, I'll review #119 and create another one if required.

I alose corrected the missing space on template.

File: templates/vault_backend_raft.j2 (hunk 1 above)
Comment (securez) [reply templates/vault_backend_raft.j2]: The spaces are fixed now. About **raft_node_id** this need to be diferent on every node/instace that can be specified via hosts_vars.

**Comment by karras** [comment]
Is it possible to merge the Raft support soon? We'd love to deploy Raft clusters with this role as soon as possible. Thanks a lot for the great effort! :)

**Pull request merged**